test/packages/parallel/aws/data_stream/ec2_metrics/_dev/deploy/tf/variables.tf

```diff
@@ -0,0 +1,22 @@
+variable "BRANCH_NAME" {
+  description = "Branch name for tagging purposes"
+  default = "unknown-branch"
+}
+
+variable "BUILD_ID" {
+  description = "Build ID in the CI for tagging purposes"
+  default = "unknown-build"
+}
+
+variable "CREATED_DATE" {
+  description = "Creation date for tagging purposes"
+  default = "unknown-date"
+}
+
+variable "ENVIRONMENT" {
+  default = "CI"
+}
+
+variable "OWNER" {
+  default = "elastic-package"
+}
```

Comment:
These tests might be executed also by developer at home, so CI and "elastic-package" won't make sense here. I suggest changing these defaults to "unknown-env", "unknown-owner".